README.md

```diff
@@ -10,6 +10,37 @@ that it can be integrated with other Juju charms in a model.
 You can deploy the operator as such:
 
 ```shell
-# Deploy the charm
+# Deploy the charm with a resource
+$ juju deploy glauth --channel edge --resource config=config.zip
+
+# Deploy the charm with no resource
 $ juju deploy glauth --channel edge
+```
+
+## Configuration
+
+In order for glauth to properly integrate with SSSD its configuration must be configured.
+
+```shell
+# LDAP domain
+juju config glauth domain=MYDOMAIN
+
+# ldap_default_bind_dn
+juju config glauth ldap-default-bind-dn=cn=serviceuser,ou=svcaccts,dc=glauth,dc=com
+
+# ldap_default_authtok
+juju config glauth ldap-password=mysecret
+
+# ldap_search_base
+juju config glauth ldap-search-base=dc=glauth,dc=com
+```
+
+The GLAuth configuration can be passed in as a resource in a *.zip. If no resource is used then a default configuration is created with no users. 
```

Comment:
The way the glauth snap is currently configured is that it will use one or more configuration files placed in a certain directory. The reason for this was so that we could be agnostic to whether there is one config or multiple.